exporter/dynatraceexporter/README.md

```diff
@@ -261,6 +263,19 @@ User should calculate this as `num_seconds * requests_per_second` where:
 
 Default: `5000`
 
+### resource_to_telemetry_conversion (Optional)
+
+When `resource_to_telemetry_conversion.enabled` is set to `true`, all resource
+attributes will be included as metric attributes.
+
+> :warning: **Please note** that the Dynatrace API has a limit of `50` attributes
+> per metric and any metric which exceeds this limit will be dropped.
+> If you think you might exceed this limit, you should use the
+> [attributes processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/attributesprocessor)
```

Comment:
Does the attributes processor actually allow "promoting" resource attributes to metric attributes?  IIRC, it just allowed modifying metric attributes.

Reply:
Ya I think you'd need to use the transform processor if you want to move resource attributes to datapoint attributes and then limit the number of attributes.

Reply:
Ah I must have been reading too quickly. Looking at the transform processor I can't see an obvious way to promote resource keys to attributes, but I am not really a TQL expert. Can you point me in the right direction?

Reply:
It would look something like:

```
set(attributes["attribute key 1"], resource.attributes["attribute key 1"])
.
.
.
.
set(attributes["attribute key N"], resource.attributes["attribute key N"])
limit(attributes, 50)
```

There isn't a function that can copy them all over at once, but seems like that could be a supported function.

Reply:
I decided to go with `keep_keys` and enable `resource_to_telemetry_conversion`

Reply:
I think _technically_ you could still end up with more than 50 attributes on data points with that technique.  If the data point has 48 attributes and you use `keep_keys` on `resource.attributes` to reduce the number of resource attributes to 3, and then use `resource_to_telemetry_conversion`, the data point will end up with 51 attributes.

Reply:
the problem with using `set` and `limit` is that dropping keys automatically is essentially impossible to do in an intelligent way. I think if users find themselves needing that level of customization they've progressed beyond something we're going to suggest in a README file and should be more comfortable with advanced collector configuration already.